### PR TITLE
Enhanced bonnie.py with disk mount on dir using partition utility

### PR DESCRIPTION
**Pull request by Naresh-ibm**
Initially script was accepting a mounted directory which involved
manual effort. Enhanced the code where accept disk and directory
names and take care of rest thing like mounting/unmounting.

Signed-off-by: Naresh Bannoth <nbannoth@in.ibm.com>

### CODE REVIEW

**Comment by Naresh-ibm** [comment]
made the changes suggested in the #302

**Comment by Naresh-ibm** [comment]
[bonnie_log_with_tearDown.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/816346/bonnie_log_with_tearDown.txt)

**Comment by narasimhan-v** [comment]
@Naresh-ibm pls change the PR name to have [v2]

File: perf/bonnie.py [hunk 1]
```diff
@@ -75,6 +89,15 @@ def test(self):
         if process.system(cmd, shell=True, ignore_status=True):
             self.fail("test failed")
 
+    def tearDown(self):
+
+        '''
+        Cleanup of disk used to perform this test
+        '''
+        self.log.info("Unmounting directory %s" % self.scratch_dir)
+        part_obj = Partition(self.disk, mountpoint=self.scratch_dir)
```
Comment (narasimhan-v) [inline perf/bonnie.py]: Why creating object again. Try to create a usable object in setUp() itself

**Comment by PraveenPenguin** [comment]
@Naresh-ibm as well please point this PR with V1 , as well closed V1 it help for reviewer

**Comment by Naresh-ibm** [comment]
@PraveenPenguin I have mentioned the PR number in my first comment. pasting the link here 
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/302

**Pull request closed without merging**